docs/user/integrate-otlp-backend/migration-to-otlp-logs.md

```diff
@@ -80,28 +112,28 @@ You can't modify an existing LogPipeline to change its output type. You must cre
          ...
    ```
 
-3. (Optional) To enrich logs with Pod labels, configure the central Telemetry resource ([Telemetry CRD](https://kyma-project.io/#/telemetry-manager/user/resources/01-telemetry)).
+1. (Optional) To enrich logs with Pod labels, configure the central Telemetry resource ([Telemetry CRD](https://kyma-project.io/#/telemetry-manager/user/resources/01-telemetry)).
 
    In contrast to a Fluent Bit LogPipeline, the `otlp` output doesn't automatically add all Pod labels. To continue enriching logs with specific labels, you must explicitly enable it in the spec.enrichments.extractPodLabels field.
 
    > [!NOTE]
    > Enrichment with Pod annotations is no longer supported.
 
-4. Deploy the new LogPipeline:
+1. Deploy the new LogPipeline:
 
    ```shell
    kubectl apply -f logpipeline.yaml
    ```
 
-5. Check that the new LogPipeline is healthy:
+1. Check that the new LogPipeline is healthy:
 
    ```shell
    kubectl get logpipeline my-otlp-pipeline
    ```
 
-6. Check your observability backend to confirm that log data is arriving.
+1. Check your observability backend to confirm that log data is arriving.
 
-7. Delete the old LogPipeline:
+1. Delete the old LogPipeline:
 
    ```shell
    kubectl delete logpipeline my-old-pipeline
```

Comment:
Confirm if this is really the name of the old pipeline; above we call it "my-http-pipeline"

Reply:
fixed